### PR TITLE
Analyze bundle size & decrease the size of main.js

### PR DESCRIPTION
**Pull request by patw0929**
* Install webpack-bundle-analyzer to get visualize size of webpack bundle file.
* Install babel-plugin-transform-react-remove-prop-types to remove react-style-proptype in production.
* Decrease main.js size from:
image
to
image

### CODE REVIEW

File: config/webpack.config.prod.js [hunk 1]
```diff
@@ -193,6 +194,18 @@ module.exports = {
...
     }),
+    new BundleAnalyzerPlugin(
+      {
+        analyzerMode: 'static',
+        reportFilename: 'report.html',
+        defaultSizes: 'parsed',
+        openAnalyzer: true,
+        generateStatsFile: false,
+        statsFilename: 'stats.json',
+        statsOptions: null,
```
Comment (nutboltu) [inline config/webpack.config.prod.js]: By default it's `null`. We can omit this

File: config/webpack.config.prod.js [hunk 2]
```diff
@@ -193,6 +194,18 @@ module.exports = {
...
       filename: 'main.css',
     }),
+    new BundleAnalyzerPlugin(
+      {
+        analyzerMode: 'static',
+        reportFilename: 'report.html',
+        defaultSizes: 'parsed',
+        openAnalyzer: true,
+        generateStatsFile: false,
+        statsFilename: 'stats.json',
```
Comment (nutboltu) [inline config/webpack.config.prod.js]: As `generateStatsFile` is `false` it wont create a file name `stats.json`

File: config/webpack.config.prod.js (hunk 1 above)
Comment (patw0929) [reply config/webpack.config.prod.js]: Got it. Fixed.

File: config/webpack.config.prod.js (hunk 2 above)
Comment (patw0929) [reply config/webpack.config.prod.js]: Removed.

File: .babelrc [hunk 3]
```diff
@@ -5,7 +5,16 @@
   ],
   "plugins": [
     "@babel/plugin-proposal-class-properties",
-    "@babel/plugin-syntax-dynamic-import"
+    "@babel/plugin-syntax-dynamic-import",
```
Comment (tomegz) [inline .babelrc]: shouldn't this be added to
   
    "env": {
      "production": {
        "plugins": [...]
      }
    }

?

Comment (patw0929) [reply .babelrc]: @tomegz Do you mean `@babel/plugin-syntax-dynamic-import` or `transform-react-remove-prop-types`? 🤔

Comment (patw0929) [reply .babelrc]: You're right. `transform-react-remove-prop-types` should be place in `production` part.
I already modified it. Thanks!

**Pull request merged**